torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -117,7 +117,7 @@ RegisterOperators reg(
      // the whole gradients in every tensor of the Autograd graph with
      // create_graph=True so we use aliasAnalysisConservative for these two OPs
      Operator(
-         "aten::backward(Tensor[](a!) tensors, Tensor?[]? grad_tensors=None, bool? retain_graph=None, bool create_graph=False) -> ()",
+         "aten::backward.TensorList(Tensor[] tensors, Tensor?[]? grad_tensors=None, bool? retain_graph=None, bool create_graph=False) -> ()",
```

Comment:
This was name duplicated before? It's serialization of calls to the unlucky variant that would break, right?

Reply:
yes but that serialization only happens for the lite interpreter where we don't have bc concerns for models doing autograd yet.